### ISSUE TITLE
Change yaw parameter when click hotspot

### ISSUE THREAD

**Issue by iwanttobefreak** [google/marzipano#465]
Hello
You have 3 images in the same direction linked with hotspot. If you go from img1 -> img2 -> img3 the orientation is correct
But if you return from img3 to img2 the initial position (yaw parameter) is oriented to img3 and you have not the walking sensation.
It's possible to change online the yaw parameter when you click one hotspot or another hotspot?

**Comment by machimoo** [comment]
Yes, this property scene.data.initialViewParameters.yaw allow you to change the initial yaw when switching to a scene

**Comment by gamebeaker** [comment]
@iwanttobefreak I had the problem and created a little tool to help me: https://github.com/gamebeaker/marzipano-hotspot-view
